### DISCUSSION TITLE
How this project integrates with ASIO, I want to use the ASIO library for network communication

### DISCUSSION THREAD

**Discussion by Mr-LZG** [PowerShell/Win32-OpenSSH#2117] [Q&A]
![image](https://github.com/PowerShell/Win32-OpenSSH/assets/22317592/1996e72e-1807-4056-b028-20f0ebc8cf31)

I'm getting some bugs like ” error C2039: 'w32_connect': Not a member of 'asio'“

**Comment by maertendMSFT** [comment]
@Mr-LZG can you give some more info on your scenario?
